requirements/test.txt

```diff
@@ -6,6 +6,8 @@ twine==3.2
 mypy>=0.920
 flake8>=3.9.2
 pre-commit>=1.0
+pytest-forked
+sklearn
```

Comment:
Why is sklearn required?

Reply:
sklearn utility is needed to run the pytest

Reply:
Can I ask where exactly? I can't find it being used.

Reply:
It will be used from datamodules.py part of test_optimization
"_SKLEARN_AVAILABLE = _module_available("sklearn")"